### ISSUE TITLE
Model Spy List tab not accessible with keyboard only

### ISSUE THREAD

**Issue by vik-chand** [eclipse-platform/eclipse.platform.ui.tools#2]
See Bug [577961](https://bugs.eclipse.org/bugs/show_bug.cgi?id=577961) - Model Spy List tab not accessible with keyboard only
